src/main/java/com/kodilla/ecommercee/UserController.java

```diff
@@ -1,4 +1,23 @@
 package com.kodilla.ecommercee;
 
+import com.kodilla.ecommercee.exceptions.UserNotFoundException;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("v1/users")
 public class UserController {
+
+    @RequestMapping(method = RequestMethod.GET, value = "getUser")
+    public GenericEntity getUser(@RequestParam Long id) throws UserNotFoundException {
+        return null;
+    }
+
+    @RequestMapping(method = RequestMethod.PUT, value = "blockUser")
+    public void blockUser(@RequestParam Long id) throws UserNotFoundException {
+    }
+
+    @RequestMapping(method = RequestMethod.GET, value = "getKey")
```

Comment:
O ile można do GET dodać body, to w praktyce nie powinno się tego robić. 
Można zostawić GET, ale w inny sposób przekazywać parametry (na przykład jako request param). Można również zmienić metodę na POST i podawać dane dalej w body. Ta druga opcja może być teraz lepsza jeżeli nie wiemy jakie dane użytkownika będą przekazywane. 
value powinno mieć wartość "generateKey"